### DISCUSSION TITLE
#23. VATZ biweekly meeting at 2022-12-07 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#313] [General]
### 1. Overall
<img width="1327" alt="image" src="https://user-images.githubusercontent.com/6308023/203641204-bf9b09c1-d61d-4748-94d0-255b5dc4cf2e.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
21 | 1/14| 7
22 | 2/17| 11

---
     
# Discussion
- CI
- Exporter
- Docker

**Comment by xellos00** [comment]
# 동요
- https://github.com/dsrvlabs/vatz/issues/312
- https://github.com/dsrvlabs/vatz/issues/314
- https://github.com/dsrvlabs/vatz/issues/319
   - https://github.com/dsrvlabs/vatz/issues/318

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/vatz/issues/326

**Comment by rootwarp** [comment]
Joonkyo

- https://github.com/dsrvlabs/vatz/issues/316
  - https://github.com/dsrvlabs/vatz/issues/298
  - https://github.com/dsrvlabs/vatz/issues/320

**Comment by kim201212** [comment]
# 형기

- https://github.com/dsrvlabs/vatz/issues/196
   - https://github.com/dsrvlabs/vatz/pull/311
- https://github.com/dsrvlabs/vatz/issues/271
- https://github.com/dsrvlabs/vatz/pull/324

**Comment by heejin-github** [comment]
## 희진
- #304 
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/23
- #327

**Comment by gnongs** [comment]
## 진홍
- https://github.com/dsrvlabs/vatz/issues/329

**Comment by meetrick** [comment]
# 황제
- 진행사항 없음

**Comment by qkrybin** [comment]
# 영빈
- https://github.com/dsrvlabs/vatz/pull/328

**Reply by xellos00** [reply]
- official plugins 1.0
  - vatz-plugins-cosmoshub  1.0
  - vatz-plugins-sysutil 1.0

**Comment by kim201212** [comment]
Logo : https://drive.google.com/drive/folders/1HPR_dKpqNIT5orrltZvTgo8Za-DOrUji

**Comment by xellos00** [comment]
Create Logo
- VATZ(Need logo)
   - Yes 
   - No
 ----
